search/src/Components/upload/upload_stepper.js

```diff
@@ -88,11 +87,12 @@ class UploadStepper extends React.Component {
         super(props);
         this.state = {
             activeStep: 0,
+            skip_mapping: {'weedcoco': -1, 'coco': -1},
```

Comment:
maybe this should be structured as each type having a list of skippable steps
```suggestion
            skip_mapping: {'weedcoco': [], 'coco': []},
```